simcon_project/conversation_templates/migrations/0001_initial.py

```diff
@@ -1,8 +1,8 @@
-# Generated by Django 3.1.3 on 2020-12-15 01:16
```

Comment:
oops i now see what u meant! Not these files, these are auto generated when you run migrations! If you could add these to the gitignore! that would be great

Reply:
https://stackoverflow.com/questions/10755655/git-ignore-tracked-files

Reply:
Okay I will do that. Thanks for posting a resource for it!

Reply:
I used the commands, but I wasn't 100% sure if the files were being ignored since they still showed up here so I just deleted them